assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -149,11 +155,17 @@ function MultiSelectionMovable({ selectedElements }) {
   // Update elements once the event has ended.
   const onGroupEventEnd = ({ targets, isRotate, isResize }) => {
     const updates = {};
+    const toRemove = [];
     targets.forEach((target, i) => {
+      const { element, updateForResizeEvent } = targetList[i];
+      if (isTargetOutOfContainer(target, pageContainer.parentNode)) {
+        removeElementFromSelection({ elementId: element.id });
```

Comment:
Deleting elements [automatically removes them from selection](https://github.com/google/web-stories-wp/blob/9852322279c814582827960472601129d60a5633/assets/src/edit-story/app/story/useStoryReducer/reducers/deleteElements.js#L95-L100). This shouldn't be necessary here.

Reply:
Yes, however, it's not deleted here yet, here it's just added to the queue for deleting later, it's for ensuring the to-be-deleted element is not being considered when making updates to Moveable.

Will test to see if it behaves as expected without it.